sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -18,6 +18,9 @@ import StudentOverawardsBalance from "@/views/student/StudentOverawardsBalance.v
 import StudentAccountApplicationInProgress from "@/views/student/StudentAccountApplicationInProgress.vue";
 import StudentAssessmentAward from "@/views/student/StudentAssessmentAward.vue";
 import StudentAppealRequests from "@/views/student/StudentAppealRequests.vue";
+import ApplicationOfferingChangeFormView from "@/views/student/application-offering-change-request/ApplicationOfferingChangeFormView.vue";
+import RequestedApplicationOfferingDetails from "@/views/student/application-offering-change-request/RequestedApplicationOfferingDetails.vue";
+import ActiveApplicationOfferingDetails from "@/views/student/application-offering-change-request/ActiveApplicationOfferingDetails.vue";
```

Comment:
As these 2 are part of tab router, probably you can create a subfolder and add them there